server/src/utils/validators.test.ts

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import SurveyDBHandler from '../databaseHandlers/surveyDBHandler';
+import { surveyResultRequestValidator } from './validators';
+
+const cosmosDBConfig = {
+  endpoint: 'https://example.org',
+  dbName: 'testingDbName'
+};
+
+describe('validators test', () => {
+  describe('testing surveyResultRequestValidator', () => {
+    describe('testing when fields is not present', () => {
```

Comment:
tiny nitpick, "testing when fields are not present"

Reply:
I will update it!